.github/actions/setup-jimm/cli-login.ts

```diff
@@ -0,0 +1,15 @@
+import { OIDC } from "../../../e2e/helpers/user/oidc";
+
+const username = process.env.USERNAME;
+const password = process.env.PASSWORD;
```

Comment:
We are no longer using these environment variables. Check: https://github.com/canonical/juju-dashboard/pull/1943#discussion_r2062910702
```suggestion
const username = process.env.ADMIN_USERNAME;
const password = process.env.ADMIN_PASSWORD;
```

Reply:
These are different from the admin user so I think it's fine to keep as different names, they're only used by this script: https://github.com/canonical/juju-dashboard/blob/main/.github/actions/setup-jimm/cli-login.ts#L3 which is used to log in so we can add the JIMM workloads controller as part of the JIMM set up.

Reply:
Understood. Thanks for the clarification!